shared_model/backend/protobuf/commands/proto_command.hpp

```diff
@@ -52,7 +64,19 @@ namespace shared_model {
 
      public:
       /// type of proto variant
-      using ProtoCommandVariantType = boost::variant<w<AddAssetQuantity>>;
+      using ProtoCommandVariantType = boost::variant<w<AddAssetQuantity>,
```

Comment:
Can be simplified with:
https://github.com/hyperledger/iroha/blob/9175e1fcc0d87fb8e61ce3a91d463e6b56841753/shared_model/backend/protobuf/transaction_responses/proto_tx_response.hpp#L51-L57